cas/apiv1/requests.go

```diff
@@ -52,20 +52,21 @@ const (
 
 // CreateCertificateRequest is the request used to sign a new certificate.
 type CreateCertificateRequest struct {
-	Template    *x509.Certificate
-	CSR         *x509.CertificateRequest
-	Lifetime    time.Duration
-	Backdate    time.Duration
-	RequestID   string
-	Provisioner *ProvisionerInfo
+	Template     *x509.Certificate
+	CSR          *x509.CertificateRequest
+	Lifetime     time.Duration
+	Backdate     time.Duration
+	RequestID    string
+	Provisioner  *ProvisionerInfo
+	IsServerCert bool
```

Comment:
I don't love this name. I feel like it can be easily confused with the existing X509 concepts of client and server certificates.

What about `IsCAServerCert`

Reply:
Sure 8bd01742